### PR TITLE
Add duration to workload command

### PR DESCRIPTION
**Pull request by isabeltomb**
Add a `--duration` option to the workload CLI command that allows a user to set the amount of time in hours that a workload should run for, this value can be provided in seconds, minutes, hours, or days. ex: 7s, 20m, 24h, 2d

### Testing:
1. Start two splinter nodes and create a circuit
2. Updload the command or smallbank smart contract
3. Start a workload using the transact CLI:
```
cargo run --manifest-path cli/Cargo.toml \
  --features=experimental workload \
  --key <private_key_file_path> \
  --workload command \
  --targets http://localhost:8080/scabbard/<circuit_id>/<service_id> \
  --target-rate 1/s \
  -vv \
  --update 2 \
  --duration 10s
```
The workload should run for the duration given and then exit, shutting down the workload.

### CODE REVIEW

File: cli/man/transact-workload.1.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ FLAGS
 
 OPTIONS
 =======
+
```
Comment (agunde406) [inline cli/man/transact-workload.1.md:44]: unrelated change in commit

File: cli/man/transact-workload.1.md [hunk 2]
```diff
@@ -85,7 +90,8 @@ transact workload \
   --update 5 \
   --smallbank-seed 10 \
   --smallbank-num-accounts 5 \
-  --targets "http://0.0.0.0:8089/scabbard/jEWSK-jdjSM/a001;http://0.0.0.0:8088/scabbard/jEWSK-jdjSM/
+  --targets "http://0.0.0.0:8089/scabbard/jEWSK-jdjSM/a001;http://0.0.0.0:8088/scabbard/jEWSK-jdjSM/ \
+  --duration 24
```
Comment (agunde406) [inline cli/man/transact-workload.1.md:94]: should this be 24h ?

File: cli/man/transact-workload.1.md (hunk 1 above)
Comment (isabeltomb) [reply cli/man/transact-workload.1.md]: Removed from commit

File: cli/man/transact-workload.1.md (hunk 2 above)
Comment (isabeltomb) [reply cli/man/transact-workload.1.md]: Forgot to update this after changing the value that duration takes, fixed now
